spec/std/system_spec.cr

```diff
@@ -18,4 +18,13 @@ describe System do
       cpu_count.should eq(shell_cpus)
     end
   end
+
+  describe "login" do
+    it "returns String or nil" do
+      shell_login = `echo $USER`.strip
+      login = System.login
+      login.should be_a(String)
```

Comment:
We typically don't check return types in spec. If you want to be sure of the returned type, annotate the method.